package.json

```diff
@@ -6,13 +6,15 @@
   "scripts": {
     "dev:start": " nodemon --exec babel-node src/index.js",
     "dev": "better-npm-run dev",
-    "test": "mocha --recursive --exit --timeout 1000000 --require @babel/register src/tests/*.test.js -watch",
+    "test": "mocha --recursive --exit --timeout 1000000 --require @babel/register src/tests/index.test.js ",
```

Comment:
And please change this script. You are referring test located in index.test.js which we don't have.

You should be reffering to every file in /src/tests/